### ISSUE TITLE
Requires to add universe section in sources.list with Ubuntu 16.10 live CD

### ISSUE THREAD

**Issue by neliom** [cw-ansible/system-install#2]
The package liblinux-lvm-perl - required at installation time - is part of the universe section, which is not in /etc/apt/sources.list of Ubuntu 16.10 liveCD (I guess that's true for other versions as well).

I suppose this is actually something for the documentation page as a tip, or did I miss anything ?

**Comment by renard** [comment]
You're right. One of the first thing I do is to setup local repositories. Thus this lead to the case you hit.

If you you have a PR you can get credit for that ;)

**Comment by neliom** [comment]
PR sent. My first, expect mistakes on the form, not on the substance :)
